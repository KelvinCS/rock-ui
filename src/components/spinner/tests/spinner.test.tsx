import { render } from '@testing-library/react';
import * as React from 'react';
import { Spinner } from '../';

test('Spinner renders correctly', () => {
  const { asFragment } = render(<Spinner />);
  expect(asFragment()).toMatchSnapshot();
});
