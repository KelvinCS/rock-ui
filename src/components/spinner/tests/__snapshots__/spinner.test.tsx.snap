// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<DocumentFragment>
  <div
    class="chakra-spinner css-1y7joxr"
  >
    <span
      class="css-f8n5zr"
    >
      Loading...
    </span>
  </div>
</DocumentFragment>
`;
